community/CCLAs/CCLA Instructions.md

```diff
@@ -0,0 +1,8 @@
+## Instructions for handling Corporate Contributor License Agreements (CCLAs)
+
+* Step 1: Download the CCLA from [here]() 
+* Step 2: Complete the form
+* Step 3: Open a PR in the https://github.com/onnx/onnx repo with the completed CCLA attached in the [CCLA directory](https://github.com/onnx/onnx/tree/master/community/CCLAs)
+* Step 4: Tag the steering committee members for review: @jspisak, @prasanthpul, @ciphr, @harryskim and Dilip Sequeira
```

Comment:
By submitting into steering committee repo, the steering committee members will automatically be notified. And we dont have to update this list each year after elections